R/plotting.R

```diff
@@ -129,14 +129,14 @@ plot_frequency <- function(seqtab, taxa, conc, neg=NULL, normalize=TRUE, showMod
     mod_melt <- do.call(rbind, mod_melts)
   }
 
-  p1 <- ggplot(data=plot_melt, aes(DNA_conc, taxon_abundance)) + xlab("DNA Concentration")
+  p1 <- ggplot(data=plot_melt, aes_string("DNA_conc", "taxon_abundance")) + xlab("DNA Concentration")
```

Comment:
Why is `aes_string` replacing `aes` here?

Reply:
Because build checks mark this with a NOTE, since the variable names used in aes, for example `DNA_conc`, are not defined in the function. This is a common problem, when using ggplot2 inside a package and calls to aes are best avoided. `aes_` and `aes_string` are dropin replacements.

https://bioconductor.org/checkResults/3.13/bioc-LATEST/decontam/malbec2-checksrc.html

```
* checking R code for possible problems ... NOTE
plot_condition: no visible binding for global variable ‘Condition’
plot_condition: no visible binding for global variable ‘DNA_conc’
plot_frequency: no visible binding for global variable ‘DNA_conc’
plot_frequency: no visible binding for global variable
  ‘taxon_abundance’
plot_frequency: no visible binding for global variable ‘Type’
plot_frequency: no visible binding for global variable ‘contam’
plot_frequency: no visible binding for global variable ‘non.contam’
Undefined global functions or variables:
  Condition DNA_conc Type contam non.contam taxon_abundance
```